index.d.ts

```diff
@@ -52,9 +52,9 @@ declare module 'io-react-native-zendesk' {
 
   interface UserInfo {
```

Comment:
To avoid type redefinition could `UserInfo` be an extension  of  `AnonymousIdentity`?

```typescript
interface UserInfo extends AnonymousIdentity{
    // user's phone
    phone?: number
    // department to redirect the chat
    department?: string
    // tags for chat
    tags?: Array<string>
  }
```

Reply:
Yes, it could. Thank you. Refactored in [4b9b6ef](https://github.com/pagopa/io-react-native-zendesk/pull/4/commits/4b9b6efff0349a6ca289e64291111d38874a6903)